src/main/java/Contacts0.java

```diff
@@ -2,15 +2,12 @@
 
 
 public class Contacts0 {
+    public static final int MAX = 100;
```

Comment:
Should be renamed to something more clear like MAX_ENTRIES

Reply:
Yes boss I will review